x-pack/plugins/secops/public/components/timeline/body/renderers/suricata_row_renderer.tsx

```diff
@@ -99,20 +108,27 @@ const DraggableValue = pure<{ data: Ecs; fieldName: string }>(({ data, fieldName
     and: [],
   };
 
+  const field = allFieldsInSchemaByName[fieldName];
+  const fieldType = field != null ? field.type : '';
+
   return (
-    <DraggableWrapper
-      key={`suricata-row-render-value-for-${fieldName}-${data._id!}`}
-      dataProvider={itemDataProvider}
-      render={(dataProvider, _, snapshot) =>
-        snapshot.isDragging ? (
-          <DragEffects>
-            <Provider dataProvider={dataProvider} />
-          </DragEffects>
-        ) : (
-          <>{`${getMappedEcsValue({ data, fieldName })}`}</>
-        )
-      }
-    />
+    <>
```

Comment:
I don't think you need the fragment here do you? It looks like root elements only below.